.idea/compiler.xml

```diff
@@ -0,0 +1,34 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project version="4">
+  <component name="CompilerConfiguration">
+    <annotationProcessing>
+      <profile name="Maven default annotation processors profile" enabled="true">
+        <sourceOutputDir name="target/generated-sources/annotations" />
+        <sourceTestOutputDir name="target/generated-test-sources/test-annotations" />
+        <outputRelativeToContentRoot value="true" />
+        <module name="crate-libs-dex" />
```

Comment:
Could this file be left out? Or what does it do?

Would be nice if there's only the main pom that's used to manage the module list

Reply:
Was added automatically by my idea, but also thought we should leave it out. I'll remove it and mark it as ignored.